middleware/dnssec/README.md

```diff
@@ -26,11 +26,17 @@ TODO(miek): think about key rollovers, and how to do them automatically.
 ~~~
 dnssec [ZONES... ] {
     key file KEY...
+    cache_capacity CAPACITY
```

Comment:
I wonder how we can make this stanza to be more in sync with *middleware/cache*? There we used `success, `denial` to indicate a category, could the same thing be done here was a term that is obvious?
`cache_capacity` is super obvious OTOH